tests/console/suseconnect_scc.pm

```diff
@@ -37,8 +37,10 @@ sub run {
     assert_script_run 'SUSEConnect --list-extensions';
 
     # add modules
-    foreach (split(',', $registration::SLE15_DEFAULT_MODULES{get_required_var('SLE_PRODUCT')} . $scc_addons)) {
-        add_suseconnect_product("sle-module-" . lc($registration::SLE15_MODULES{$_}));
+    if (is_sle && sle_version_at_least('15')) {
+        foreach (split(',', $registration::SLE15_DEFAULT_MODULES{get_required_var('SLE_PRODUCT')} . $scc_addons)) {
```

Comment:
Hm. IMHO it would be better if we have a SLE version specific hash so that SLE12 works as well. Would you like to try? If not we can also accept as is.

Reply:
I don't think I understand what you mean by a hash here...

Reply:
Sorry, I confused stuff here. Of course there are no default modules for SLE12 so your code is valid as is.